### ISSUE TITLE
Is it possible or going to be possible to use the RAT from mac operating system.

### ISSUE THREAD

**Issue by DemonSpeed** [SaherBlueEagle/XPR-2020-Free#4]
Is it possible or going to be possible to use the RAT from mac operating system. What I mean by that is it possible to create a payload RAT from mac and use the controls.

**Comment by SaherBlueEagle** [comment]
yes sure , the next version of the jRAT  will be XPR  and controller itself can run from Windows , MAC , Linux , as the SBE-jRAT
https://www.facebook.com/watch/?v=646241972734504
